modules/ROOT/pages/database-administration/composite-databases/manage-composite-databases.adoc

```diff
@@ -0,0 +1,170 @@
+
+.Query
+[source, cypher]
+----
+CREATE COMPOSITE DATABASE inventory IF NOT EXISTS
+----
+
+This will not create a new composite database, because a composite database with the name `inventory` already exists.
+
+Second, adding `OR REPLACE` to the command will result in any existing database being deleted and a new one being created.
+
+.Query
+[source, cypher]
+----
+CREATE OR REPLACE COMPOSITE DATABASE inventory
+----
+
+This is equivalent to running `DROP DATABASE inventory IF EXISTS` followed by `CREATE COMPOSITE DATABASE inventory`.
+
+The behavior of `IF EXISTS` and `OR REPLACE` apply to both standard and composite databases (e.g. a composite database may replace a standard database or another composite database).
+
+[NOTE]
+====
+The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used together.
+====
+
+
+[[composite-databases-stop]]
+== Stop composite databases
+
+Databases can be stopped using the command `STOP DATABASE`.
+
+.Query
+[source, cypher]
+----
+STOP DATABASE inventory
+----
+
+[NOTE]
+====
+Both standard databases and composite databases can be stopped using this command.
+====
+
+The status of the stopped database can be seen using the command `SHOW DATABASE name`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE inventory YIELD name, requestedStatus, currentStatus
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------+
+| name        | requestedStatus | currentStatus |
++-----------------------------------------------+
+| "inventory" | "offline"       | "offline"     |
++-----------------------------------------------+
+----
+
+[[composite-databases-start]]
+== Start composite databases
+
+Databases can be started using the command `START DATABASE`.
+
+.Query
+[source, cypher]
+----
+START DATABASE inventory
+----
+
+[NOTE]
+====
+Both standard databases and composite databases can be started using this command.
+====
+
+The status of the started database can be seen using the command `SHOW DATABASE name`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE inventory YIELD name, requestedStatus, currentStatus
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="3*<m"]
+----
++-----------------------------------------------+
+| name        | requestedStatus | currentStatus |
++-----------------------------------------------+
+| "inventory" | "online"        | "online"      |
++-----------------------------------------------+
+----
+
+[[composite-databases-delete]]
+== Delete composite databases
+
+Composite databases can be deleted using the command `DROP COMPOSITE DATABASE name`.
+This command will fail if the targeted database is not a composite database.
+It is also possible to use the shorter command `DROP DATABASE name` for composite databases, but this command will drop any database and not fail in case the targeted database is not a composite database.
```

Comment:
looking at the preview this block of text is a bit dense and covers 2 things. I wonder if it would be nicer to have line 163 (about `drop database`) after the query example instead?

Reply:
Good suggestion 👍